src/hotspot/share/jfr/leakprofiler/checkpoint/objectSampleCheckpoint.cpp

```diff
@@ -92,6 +91,15 @@ void ObjectSampleCheckpoint::on_thread_exit(JavaThread* jt) {
   }
 }
 
+void ObjectSampleCheckpoint::clear() {
+  assert(SafepointSynchronize::is_at_safepoint(), "invariant");
+  if (unloaded_thread_id_set != NULL) {
+    delete unloaded_thread_id_set;
+    unloaded_thread_id_set = NULL;
+  }
+  assert(unloaded_thread_id_set == NULL, "invariant");
+}
+
```

Comment:
This does not seem to be called anywhere. Is it expected?

Reply:
Thanks Jaroslav, changes were lost when moving this from main to 17.